styles.css

```diff
@@ -0,0 +1,64 @@
+html,
+  body {
+  min-width: 290px;
+  color: #4d4e53;
+  background-color: #ffffff;
+  font-family: 'Open Sans', Arial, sans-serif;
+  line-height: 1.5;
+}
+  #navbar {
+  position: fixed;
+  min-width: 290px;
+  top: 0px;
+  left: 0px;
+  width: 300px;
+  height: 100%;
+  border-right: solid;
+  border-color: rgba(0, 22, 22, 0.4);
+  background-color: aquamarine;
+}
+
```

Comment:
I think it would be better to be careful on line spaces in both, html and css

Reply:
- [ ] Your appoint it's good, but remember that you need to clarify why you are requesting these changes, in this way the student will learn and reduce his frustration

Reply:
* As pointed by the previous reviewer Kindly bear in mind that it is always better to clearly state why a change is required.
A more well-explained comment will be similar to:
_ Why don't you use the <main> tag instead of a div, that would improve the HTML structure and make the page more accessible to browsers and be more meaningful to developers_

Reply:
Can you check it now? 😣😣😣